QuickStarts/Finding Fraudulent Communities with Neo4j/Louvain-Community-Detection.md

```diff
@@ -136,7 +151,9 @@ CREATE OR REPLACE VIEW p2p_users_vw (nodeId) AS
 ```
 <!-- ------------------------ -->
 
-## Step 4: Running your Algorithms
+## Running your Algorithms
+Duration: 10
```

Comment:
Please double check app name. Neo4j_GDS_app and neo4j_graph_analytics are used simultaneously

Reply:
Settled on the neo4j_graph_analytics name.